packages/core/docs/.vuepress/config.js

```diff
@@ -183,141 +132,173 @@ module.exports = {
-                    children: [
-                        ['/shopify/checkout', 'Checkout']
-                    ],
-                }
+        {
+            title: 'Essentials',
+            collapsable: false,
+            children: [
+                ['/shopify/', 'Introduction'],
+                ['/shopify/getting-started', 'Getting Started'],
+                ['/shopify/configuration', 'Configuration'],
+                ['/shopify/authorization-strategy', 'Authorization'],
+                ['/shopify/feature-list', 'Feature list'],
+                ['/shopify/maintainers', 'Maintainers and support']
+            ]
+        },
+        {
+            title: 'Composables',
+            collapsable: false,
+            children: [
+                ['/shopify/use-product', 'useProduct'],
+                ['/shopify/use-user', 'useUser'],
+                ['/shopify/use-user-order', 'useUserOrder'],
+                ['/shopify/use-cart', 'useCart'],
+                ['/shopify/use-category', 'useCategory'],
+                ['/shopify/use-content', 'useContent'],
+                ['/shopify/use-search', 'useSearch'],
+            ]
+        },
+        {
+            title: 'API Client',
+            collapsable: false,
+            children: [
+                ['/shopify/api-client-reference', 'Methods reference']
+            ]
+        },
+        {
+            title: 'Theme',
+            collapsable: false,
+            children: [
+                ['/shopify/auth-middleware', 'Auth Middleware']
             ],
+        },
+        {
+            title: 'Other',
+            collapsable: false,
+            children: [
+                ['/shopify/checkout', 'Checkout']
+            ],
+        }
+      ],
       '/': [
         {
-          title: 'Getting started',
+          title: '',
           collapsable: false,
           children: [
-            ['/', 'Introduction'],
-            ['/general/where-to-start', 'Where to start?'],
-            ['/general/installation', 'Installation'],
-            ['/general/key-concepts', 'Key concepts'],
-            ['/general/enterprise', 'Enterprise'],
-            ['/general/support', 'Support']
+            ['/', '👋 Meet Vue Storefront'],
+            ['/general/installation', '💻 Installation'],
+            ['/integrations/', '🔌 Integrations'],
+            ['/general/enterprise', '🚀 Enterprise'],
+            ['/general/support', '🙋 Support'],
           ]
         },
         {
-          title: 'Guides',
+          title: 'Getting started',
           collapsable: false,
           children: [
+            ['/general/key-concepts', 'Key concepts'],
             ['/guide/theme', 'Theme'],
             ['/guide/configuration', 'Configuration'],
+            ['/advanced/internationalization', 'Internationalization'],
+            ['/advanced/logging', 'Logging'],
+            // ['/', 'Glossary']
+          ]
+        },
+        {
+          title: 'Composition',
```

Comment:
How about changing this title to "Essential elements" or "Essential ingredients", would it be better describing name for this section?

Reply:
I think `Composition` better explains what's inside of this category. `Essentials` or `Essential elements` is too generic and similar to `Getting started` since it also contains things essential to working with VSF projects.

WDYT @igorwojciechowski and @lukaszjedrasik?

Reply:
For me `Composition` is cleaner and sounds cool 😄. Even in the vue3 docs, we have a Reusability & `Composition` section.

Reply:
that's okay let's leave 'Composition' :)